### DISCUSSION TITLE
플레이 했을 때 가끔 머리카락이 이상해요. (Character Control)

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#229] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/97988822/150087145-a15c355a-6ebf-4977-bd16-b9493ef4449a.png)
![image](https://user-images.githubusercontent.com/97988822/150087219-c0035fa6-1002-4469-affd-68499fa6862d.png)
![image](https://user-images.githubusercontent.com/97988822/150087282-fbb3ac43-e97c-42f3-a431-dbf8d60a0308.png)

매번 발생하는 건 아니구요. 가끔 발생하는 이슈입니다. 보통은 멀쩡한 헤어스타일입니다. 동영상도 찍어놨는데 동영상은 첨부가 안되네요. ^^;

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전 1.1.5 에서도 지속적으로 나타나는지 확인 부탁드립니다.
